pkg/metrics/prometheus_exporter.go

```diff
@@ -0,0 +1,33 @@
+package metrics
+
+import (
+	"fmt"
+	"net/http"
+
+	"contrib.go.opencensus.io/exporter/prometheus"
+	log "github.com/Azure/aad-pod-identity/pkg/logger"
+)
+
+// newPrometheusExporter creates prometheus exporter and run the same on given port
+func newPrometheusExporter(namespace string, portNumber string, log log.Logger) error {
```

Comment:
Don't we need to return `view.Exporter` here and register that exporter using `view.RegisterExporter`?

Reply:
Done. It was working earlier because by default Collected data will be reported via all the registered exporters.